### PR TITLE
airoha: disable swconfig

### PR DESCRIPTION
**Pull request by abajk**
Swconfig isn't used by this target and can be disabled for this reason. Airoha doesn't even have an Ethernet driver. In the future, this target should get a DSA driver.

Signed-off-by: Aleksander Jan Bajkowski <olek2@wp.pl>

### CODE REVIEW

**Review by hauke — APPROVED** [review]

File: target/linux/airoha/config-5.15 [hunk 1]
```diff
@@ -173,6 +169,8 @@ CONFIG_MTD_UBI_BLOCK=y
 CONFIG_MTD_UBI_WL_THRESHOLD=4096
 CONFIG_MUTEX_SPIN_ON_OWNER=y
 CONFIG_NEED_DMA_MAP_STATE=y
+CONFIG_NET_FLOW_LIMIT=y
+CONFIG_NET_SELFTESTS=y
```
Comment (Ansuel) [inline target/linux/airoha/config-5.15:173]: smell like generic config?

Comment (abajk) [reply target/linux/airoha/config-5.15:173]: CONFIG_NET_FLOW_LIMIT is enabled by CONFIG_SMP.
CONFIG_NET_SELFTESTS is enabled by CONFIG_NET_DSA.
Many targets support SMP or DSA therefore these symbols are enabled there.
There are subtargets without DSA and SMP e.g. at91, lantiq/ase, lantiq/xway, lantiq/xway_legacy, realtek/rtl838x, uml....

Comment (Ansuel) [reply target/linux/airoha/config-5.15:173]: Sorry for the delay... wonder if selftests is needed?

**Comment by hauke** [comment]
Thank you for the patches, I applied them to master in d81e948dc2d9b45e6b65fc8d10badb791783d7f7 and bb2fb4f16aef574921cb75e2205b2a3c38822588.

**Pull request closed without merging**